### ISSUE TITLE
Add `[End]` to list of keywords to clean in knowledge pipeline config

### ISSUE THREAD

**Issue by abhi1092** [instructlab/sdg#399]

**Comment by bbrowning** [comment]
Just to have a bit more context here, is this because we've observed the model output varies its capitalization when generating responses? I assume so, but don't want to assume incorrectly.

**Comment by abhi1092** [comment]
Yup.  The model tends to sometimes change the capitalization. So we simply cover both usecases
